tests/unit/providers/wikipedia/backends/test_elastic.py

```diff
@@ -1,86 +1,59 @@
-"""Unit tests for the Merino v1 suggest API endpoint for the Wikipedia provider."""
+"""Unit tests for the Merino v1 suggest API endpoint for the Elastic Backend."""
+from typing import Optional
 from unittest.mock import AsyncMock
 
 import pytest
 from elasticsearch import AsyncElasticsearch
 from pytest_mock import MockerFixture
 
-from merino.config import settings
 from merino.exceptions import BackendError
-from merino.providers.wikipedia.backends.elastic import SUGGEST_ID, ElasticBackend
-from merino.providers.wikipedia.backends.fake_backends import FakeEchoWikipediaBackend
-from merino.providers.wikipedia.provider import (
-    ADVERTISER,
-    ICON,
-    Provider,
-    WikipediaSuggestion,
+from merino.providers.wikipedia.backends.elastic import (
+    SUGGEST_ID,
+    ElasticBackend,
+    ElasticBackendError,
 )
-from tests.unit.types import SuggestionRequestFixture
 
-
-@pytest.fixture(name="wikipedia")
-def fixture_wikipedia() -> Provider:
-    """Return a Wikipedia provider that uses a test backend."""
-    return Provider(backend=FakeEchoWikipediaBackend())
+FAKE_ELASTIC_CLOUD_ID = (
+    "cluster:dXMtZWFzdC0xLmF3cy5mb3VuZC5pbyQ0ZmE4ODIxZTc1NjM0MDMyYmVk"
+    "MWNmMjIxMTBlMmY5NyQ0ZmE4ODIxZTc1NjM0MDMyYmVkMWNmMjIxMTBlMmY5Ng=="
+)
 
 
 @pytest.fixture(name="es_backend")
 def fixture_es_backend() -> ElasticBackend:
     """Return an ES backend instance."""
     return ElasticBackend(
-        cloud_id=settings.providers.wikipedia.es_cloud_id,
-        user=settings.providers.wikipedia.es_user,
-        password=settings.providers.wikipedia.es_password,
+        url="http://localhost:9200",
+        cloud_id=None,
     )
 
 
-def test_enabled_by_default(wikipedia: Provider) -> None:
-    """Test for the enabled_by_default method."""
-    assert wikipedia.enabled_by_default
-
+def test_initialize_es_backend_cloud_id_success():
+    """Test that the ElasticBackend initializes successfully with a cloud_id."""
+    es = ElasticBackend(cloud_id=FAKE_ELASTIC_CLOUD_ID)
+    assert es
```

Comment:
Since units are white-box tests, we can spy on the AsyncElasticsearch object to make sure it's being initialized as expected:

```suggestion
def test_initialize_es_backend_cloud_id_success(mocker: MockerFixture) -> None:
    """Test that the ElasticBackend initializes successfully with a cloud_id."""
    es_client_spy: Any = mocker.spy(AsyncElasticsearch, "__init__")

    ElasticBackend(cloud_id=FAKE_ELASTIC_CLOUD_ID)

    es_client_spy.assert_called_once()
    assert es_client_spy.call_args.kwargs["cloud_id"] == FAKE_ELASTIC_CLOUD_ID
```